core/templates/dev/head/domain/topic/TopicObjectFactory.js

```diff
@@ -85,36 +88,6 @@ oppia.factory('TopicObjectFactory', ['SubtopicObjectFactory',
       return null;
     };
 
-    /**
-     * @param {number} id - The id of the subtopic that was deleted.
-     * @param {string} title - The title of the subtopic that was deleted.
-     * @param {array(String)} skillIdsForSubtopic - The skillIds array of the
-     * deleted subtopic.
-     * @param {boolean} isNewlyCreated - Whether the subtopic to be deleted was
-     * created in the same changelist as it was deleted.
-     */
-    Topic.prototype.undoDeleteSubtopic = function(
```

Comment:
Interesting ... was this just unused?

Reply:
Yeah, since an error is thrown when trying to undo a delete subtopic command, this function is now not used.